common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2057,6 +2060,22 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("2g")
 
+  val FILE_CACHE_MAX_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.shuffle.cache.max")
+      .categories("worker")
+      .doc("Max size for shuffle file cache to memory")
+      .version("0.4.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("10g")
```

Comment:
10g is too big, for some users' environments total direct memory is below 10g. maybe 512m is better